src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -708,6 +708,25 @@ public enum BloodhoundCoverageUpdateMode {
   @Option("Maximum length of Strings in generated tests")
   public static int string_maxlen = 1000;
 
+  /**
+   * Constructs missing inputs on demand. By default, when calling a method, Randoop uses as
+   * arguments whatever values Randoop has already generated. This may prevent Randoop from calling
+   * a method, if Randoop has not yet generated any values of the appropriate type or if the user
+   * did not specify that type on the command line. With demand-driven input creation, Randoop
+   * immediately attempts to construct inputs for the method under test.
+   */
+  @Option("Construct method inputs on demand, if no value exists yet of the given type")
```

Comment:
Whether a value already exists is irrelevant (though it's what I incorrectly believed in the past!).  It's about whether it is possible to create a value as the return value of a method in the SUT.